### PR TITLE
Add responsive viewport width meta tag.

### PR DESCRIPTION
**Pull request by LearnWebCode**

### CODE REVIEW

**Comment by drgfunk** [comment]
@aaronbushnell Should we bring this up at our annual committee meeting?

**Comment by aaronbushnell** [comment]
@drgfunk WHO IS THIS

**Comment by drgfunk** [comment]
I'm simply known as "The Chairman."

File: app/templates/_index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (aaronbushnell) [inline app/templates/_index.html]: Let's drop the `.0` from `1.0`. Just using `1` will work here!

**Comment by LearnWebCode** [comment]
@aaronbushnell Good call!

**Pull request merged**